gsuite_reports_rules/gsuite_drive_visibility_change.py

```diff
@@ -3,9 +3,18 @@
 
 
 def rule(event):
+    # This Tuple refers to events where documents have changed perms due to parent folder change
+    inheritance_events = (
+        "change_user_access_hierarchy_reconciled",
+        "change_document_access_scope_hierarchy_reconciled",
+    )
+
     if deep_get(event, "id", "applicationName") != "drive":
         return False
 
+    if deep_get(event, "events", "name") in inheritance_events:
```

Comment:
perhaps add a comment above this explaining why we are skipping/ignoring the inheritance events?

Reply:
Added